src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14159265359;
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double calculateArea() {
+        return PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: "
+                + calculateArea() + " sq. units, "
+                + "radius: " + radius + " units, "
+                + "color: " + color + ".";
+    }
```

Comment:
contradicts task
behavior - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() **(you shouldn't override the toString() method for this)**.

Please implement draw() method instead

Reply:
Sorry, I've read 'shouldn't' as 'should' ((